src/com/dabomstew/pkrandom/newgui/NewRandomizerGUI.java

```diff
@@ -1987,6 +1991,9 @@ private void initialState() {
         peChangeImpossibleEvosCheckBox.setVisible(true);
         peChangeImpossibleEvosCheckBox.setEnabled(false);
         peChangeImpossibleEvosCheckBox.setSelected(false);
+        ptIsDualTypeCheckBox.setVisible(true);
```

Comment:
Could you move these up to be next to the `ptFollowMegaEvosCheckBox` block on line 1939?